core/src/main/scala/org/apache/spark/deploy/history/ApplicationCache.scala

```diff
@@ -410,34 +409,25 @@ private[history] class CacheMetrics(prefix: String) extends Source {
     ("update.triggered.count", updateTriggeredCount))
 
   /** all metrics, including timers */
-  private val allMetrics = counters ++ Seq(
+  private val allMetrics: Seq[(String, Metric with Counting)] = counters ++ Seq(
```

Comment:
Either I was just being explicit about what came in, or the IDE decided to get involved. Removed